ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -35,26 +35,46 @@ def toProtocolElement(self):
 
 
 class SimulatedPhenotypeAssociationSet(AbstractPhenotypeAssociationSet):
-    def __init__(self, parentContainer, localId, randomSeed):
+    def __init__(self, parentContainer, localId, randomSeed,
+                 numAssociations=2):
```

Comment:
@dcolligan  [commit 6a3d879](https://github.com/ga4gh/server/pull/1348/commits/6a3d879845982419c2481e7fcc5f308cb749be19) should address this.  Thanks, -b